libparsec/crates/client_connection/src/constant.rs

```diff
@@ -0,0 +1,4 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+pub(crate) const CLIENT_USER_AGENT: &str =
+    concat!("Parsec-Client/", std::include_str!("../../../../version"));
```

Comment:
Why not putting this in [lib.rs](https://github.com/Scille/parsec-cloud/blob/998ef300224e63f14d5a805b075c9dfc25e81fd8/libparsec/crates/client_connection/src/lib.rs)? (That would be consistent with other constants like `PARSEC_CONTENT_TYPE` and `API_VERSION_HEADER_NAME`)

Reply:
done